.github/workflows/gradle-wrapper-validation.yaml

```diff
@@ -12,4 +12,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: gradle/wrapper-validation-action@v2.1.3
\ No newline at end of file
+      - uses: gradle/actions/wrapper-validation@v3
```

Comment:
I usually prefer pinned versions and using [dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates) to open PRs upgrading it.
So I know before merging whether the CI is happy or not.
Using `@v3` will always use the latest patch or minor and might break things requiring immediate action.

Reply:
Renovate does that already btw eg https://github.com/open-telemetry/opentelemetry-android/pull/303

Reply:
Yeah, we pin exact versions in our distro. I guess I can do that here as well. Thanks!